src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -2312,6 +2318,72 @@ void DoorLockServer::sendGetYearDayScheduleResponse(chip::app::CommandHandler *
     commandObj->AddResponse(commandPath, response);
 }
 
+bool DoorLockServer::holidayIndexValid(chip::EndpointId endpointId, uint8_t holidayIndex)
+{
+    uint8_t holidaysSupported;
+    if (!GetNumberOfHolidaySchedulesSupported(endpointId, holidaysSupported))
+    {
+        return false;
+    }
+
+    // appclusters, 5.2.4.22-25: year day index changes from 1 to maxNumberOfHolidaySchedules
```

Comment:
What does this have to do with "year day index"?

Reply:
Just a typo, I copy pasted the comment, here's the result. I should probably revise all the comments.